launcher/mainwindow_moc.cpp

```diff
@@ -236,16 +236,32 @@ void MainWindow::on_aboutButton_clicked()
 void MainWindow::updateTranslation()
 {
 #ifdef ENABLE_QT_TRANSLATIONS
-	const std::string translationFile = settings["general"]["language"].String() + ".qm";
-	logGlobal->info("Loading translation '%s'", translationFile);
+	const std::string translationFile = settings["general"]["language"].String()+ ".qm";
+	QString translationFileResourcePath = QString{":/translation/%1"}.arg(translationFile.c_str());
 
-	if (!translator.load(QString{":/translation/%1"}.arg(translationFile.c_str())))
+	logGlobal->info("Loading translation %s", translationFile);
+
+	if(!QFile::exists(translationFileResourcePath))
+	{
+		logGlobal->warn("Translation file %s does not exist", translationFileResourcePath.toStdString());
```

Comment:
Probably no need to show this in console
```suggestion
		logGlobal->debug("Translation file %s does not exist", translationFileResourcePath.toStdString());
```